x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_embeddable_chart/timeseriesexplorer_embeddable_chart.js

```diff
@@ -283,15 +289,13 @@ export class TimeSeriesExplorerEmbeddableChart extends React.Component {
       )
       .pipe(
         map((resp) => {
-          const { mlJobService } = this.context.services.mlServices;
           const anomalies = resp.anomalies;
-          const detectorsByJob = mlJobService.detectorsByJob;
           anomalies.forEach((anomaly) => {
             // Add a detector property to each anomaly.
             // Default to functionDescription if no description available.
             // TODO - when job_service is moved server_side, move this to server endpoint.
-            const jobId = anomaly.jobId;
-            const detector = get(detectorsByJob, [jobId, anomaly.detectorIndex]);
+            const jobDetectors = get(selectedJob.analysis_config, 'detectors', []);
```

Comment:
```suggestion
            const jobDetectors = selectedJob.analysis_config.detectors;
```
This is a good opportunity to remove `get`. 
Also the detectors list will never be missing.

Reply:
Updated in 53b573ff6d8d57acfe1ca468df74f5a8d63354a3